### DISCUSSION TITLE
"421: Misdirected Request" when connecting remotely

### DISCUSSION THREAD

**Discussion by dobrynski** [transmission/transmission#1941] [Q&A]
`421: Misdirected RequestTransmission received your request, but the hostname was unrecognized.To fix this, choose one of the following options:Enable password authentication, then any hostname is allowed.Add the hostname you want to use to the whitelist in settings.If you're editing settings.json, see the 'rpc-host-whitelist' and 'rpc-host-whitelist-enabled' entries.This requirement has been added to help prevent DNS Rebinding attacks.`

I've updated many times before without an issue. There has always been a user/pass set.

I've not been able to figure out what the problem is.

**Comment by GurbEHV** [comment]
I'm experiencing the same.
In settings.json, it shows rpc-authentication-required as false, even though I have always used a username and password. If I stop the container, edit settings.json with user/pass/authentication=true  again and restart the container...it overwrites settings.json immediately but again with authenthication set to false again, with a blank username, but with a hash for the password.

**Comment by dobrynski** [comment]
I also always used a username and password though I don't remember how I originally set it.

I found the solution was to re-create my container with -e USER=username -e PASS=password.

I think the upstream Transmission error message does not reflect the change that the container requires. It should probably be mentioned somewhere in the README that USER and PASS are no longer optional.

**Comment by GurbEHV** [comment]
Thanks, that seems to work!
